### ISSUE TITLE
Screen reader doesn't read all keyboard shortcut punctuation 

### ISSUE THREAD

**Issue by e218736** [jupyterlab/jupyterlab#15497]
### Problem

When navigating using a screen reader (Windows Narrator) some keyboard shortcuts that include punctuation are not read out correctly. 

For example, within keyboard shortcuts 
Activate Previous Tab Bar	<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>,</kbd> 

Screen reader annouces 'Activate Previous Tab Bar control plus shift plus' and ignores the comma

### Proposed Solution

Adding aria-labels and aria-keyshortcuts properties to announce specific key combinations properly

**Comment by JasonWeill** [comment]
@e218736 Thank you for opening this! In which context do you encounter this issue? Navigating the pull-down menus, command palette, focusing toolbar buttons, or somewhere else?

**Comment by e218736** [comment]
Hi @JasonWeill, 
whilst navigating through the top menu bar e.g. within Tabs, 3 of 5 keyboard shortcuts are not read out correctly.
Also when opening keyboard shortcuts from the help menu any shortcuts with punctuation are announced incorrectly.
Currently working on a fix for this with aria-labels and will check whether command palette & toolbar buttons should be included.
